### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by Bowrna**
Mounting config in docker way of running airflow

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/howto/docker-compose/index.rst [hunk 1]
```diff
@@ -101,6 +101,7 @@ Some directories in the container are mounted, which means that their contents a
 
 - ``./dags`` - you can put your DAG files here.
 - ``./logs`` - contains logs from task execution and scheduler.
+- ``./config`` - you can add custom log parser or add ``airflow_local_settings.py`` to configure cluster policy.
```
Comment (potiuk) [inline docs/apache-airflow/howto/docker-compose/index.rst:104]: Hm. But I believe config is not mounted :

```
  volumes:
    - ${AIRFLOW_PROJ_DIR:-.}/dags:/opt/airflow/dags
    - ${AIRFLOW_PROJ_DIR:-.}/logs:/opt/airflow/logs
    - ${AIRFLOW_PROJ_DIR:-.}/plugins:/opt/airflow/plugins
```

Comment (Bowrna) [reply docs/apache-airflow/howto/docker-compose/index.rst:104]: oops i thought its missed out in documentation part alone. could i add that as part of the docker-compose.yml file too @potiuk ?

Comment (Bowrna) [reply docs/apache-airflow/howto/docker-compose/index.rst:104]: @potiuk could you let me know if i can add as part of docker-compose.yml? bringing this up as a gentle reminder

Comment (pankajastro) [reply docs/apache-airflow/howto/docker-compose/index.rst:104]: @potiuk I think it mounted here https://github.com/apache/airflow/blob/90ee7639478bc8534164c4bbb05a1ccfb04272dd/docs/apache-airflow/howto/docker-compose/docker-compose.yaml#L75

or do we need to mount it at some other places too?

Comment (potiuk) [reply docs/apache-airflow/howto/docker-compose/index.rst:104]: config?  @pankajastro ?

Comment (potiuk) [reply docs/apache-airflow/howto/docker-compose/index.rst:104]: > @potiuk could you let me know if i can add as part of docker-compose.yml? bringing this up as a gentle reminder

Yes. you can.

Comment (pankajastro) [reply docs/apache-airflow/howto/docker-compose/index.rst:104]: > config? @pankajastro ?

yeah it missing 🤦‍♂️

**Comment by Bowrna** [comment]
I made some mess with committing the changes into the branch. my apologies. I will create another pull request and submit the changes as I am worried that this may involve many people reviewing the changes.

**Pull request closed without merging**